src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -695,5 +695,77 @@ void PostStepsList::add_convert_layout_impl(const std::vector<uint64_t>& dims) {
         },
         "convert layout " + vector_to_string(dims));
 }
+
+void PostStepsList::add_convert_color_impl(const ColorFormat& dst_format) {
+    m_actions.emplace_back(
+        [dst_format](const Output<Node>& node, PostprocessingContext& context) {
+            if (context.color_format() == dst_format) {
+                return std::make_tuple(node, false);
+            }
+            if ((context.color_format() == ColorFormat::RGB || context.color_format() == ColorFormat::BGR) &&
```

Comment:
Conver to 
if()
else if()
else

as each case is excluding

Reply:
Agree your idea!

Reply:
I mean something  like:
```cpp
        [dst_format](const Output<Node>& node, PostprocessingContext& context) {
            if (context.color_format() == dst_format) {
                return std::make_tuple(node, false);
            } else if ((context.color_format() == ColorFormat::RGB || context.color_format() == ColorFormat::BGR) &&
                (dst_format == ColorFormat::RGB || dst_format == ColorFormat::BGR)) {
                auto res = reverse_channels({node}, context);
                context.color_format() = dst_format;
                return res;
            } else{
                OPENVINO_THROW("Source color format '",
                            color_format_name(context.color_format()),
                            "' is not convertible to '",
                            color_format_name(dst_format),
                            "'");
           }
        }
```
Is detail maybe not worthy to apply some time excluding case optimize better.

Reply:
oh, I have misunderstood your suggestion, will update it.